tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/balancing/ElasticQueueTest.java

```diff
@@ -398,7 +407,7 @@ private void prepareNodesAndStartCombinedHeadTail() throws Exception {
          configuration.setName("Node" + nodeId);
          configuration.setBrokerInstance(new File(getTestDirfile(), configuration.getName()));
          configuration.addAcceptorConfiguration("tcp", "tcp://localhost:" + (base_port + (nodeId)) + "?redirect-to=" + balancerConfigName + ";amqpCredits=1000;amqpMinCredits=300");
-         nodes.add(new EmbeddedActiveMQ().setConfiguration(configuration));
+         nodes.add(new EmbeddedActiveMQ().setConfiguration(configuration).setPropertiesResourcePath(null));
```

Comment:
This seems odd...is it actually necessary? Is there a default file on the classpath screwing with this test in some way? That would seem like a general issue needing fixed in itself (also against ARTEMIS-3627 perhaps?), it really doesnt feel like unrelated individual tests should need to be randomly setting this null to work.

Reply:
It does need a way to turn it off so setting null is good. 
Yes there is a file on the classpath, that changed the jmx name, I will fix that in another commit

Reply:
Having a way to do it is fine, but it doesnt seem like this test (or mostly any other) should need to do so.

Reply:
agree. thanks.